src/pipes/ellipsis.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe } from '@angular/core';
+
+@Pipe({
+  name: 'ellipsis'
+})
+export class EllipsisPipe {
```

Comment:
We should always be running the HTML regex over the input, and then we should check the length of the new preview:

```ts
const withoutHtml = str.replace(...);

if (str.length > 250) {
  // return sliced
}

// return
```